### PR TITLE
Update 30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea434849…

### PR DESCRIPTION
**Pull request by cardanocasinovip**
…50.yaml

$CHIP token Categories and Treasury Address Update

### CODE REVIEW

File: src/tokens/30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea43484950.yaml [hunk 1]
```diff
@@ -16,10 +16,10 @@ maxSupply: '1600000000000000'
...
-  - addr1q8e2p26qtrz02eygsgm65d04afzfwqgn0xccq3ah6haxj4tl28ezl6703wu7fhn9hax7yyp6466hn6pnamrwjenhy4cs3hmr50
-  - addr1q8rqmar56e9t05vueh5n69gdnnnfguv9m5jtvjltc6p4zu9x6n7x0yn54ckn8qwlk75pm8ewhck8kkfd56devu2lygastz5pgq
-  - addr1q93zlwt4cxge5p8fhdczt8qwrva0vuy3h88yu3lzvs55j7lsw8u030madzgeflqhsevrzdp4nls3gd0g47nef8ffj02sa7gr0t
-  - addr1qyy86us6kfrg3rsy0nhssrlkjyq4yw8wasn0n8qgk9gqp0lz3kqj9ut6p5c4gxlg7vehhkzsyfejj0gkpyf7w6d20ffqt2et79
+  - addr1qypez7msuwynwjm77vtrmmn9pm5v22y5th26exxe0kepst24k83f37z3s7fapxjamxkeedc085vg0y34zgn8x4c5m05q5nzhsy
+  - addr1q9v5l0dmy6jaz9dwkww9ghtg3yzfr6js2pejwyp6m7tjtk87hpufs2s7rtta3x4tfwa79qfht3ccxhwd7p835jytpa4qf2y00d
+  - addr1q90e9f6p69ad95998f5l9huuf70s3pd3m7yg9xynuypln3sr96ajdrxwdjdxrkayjn48qm2h35jxjdnsqlusfg027kyq97xapw
+  - addr1q9nczapcdhg7yc23ysun3tjtkx2lydkjw76pem86dw7htu69sqkqf5r066h5ca2g5qldne4yn8r3hxzm73ez6urgss3sruu6l2
+  - addr1q8swgmuz39jpknze9q8ljmwq6ndse8wpp36d8xfcxf7vkj5qp8z9c4tw4tkxer2dlsejueexww9jdp867de9xqvw6ussslysgu
+  - addr1q8v6xff20j88n7jekxw53x0g3pp7c2gcsx738q456hue6m0u9w6rhn6wgwqpsg24jfnk79lexhuzpmaj4rf9ghtw73jqmp0e04
```
Comment (0xj4m35) [inline src/tokens/30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea43484950.yaml:24]: @cardanocasinovip your PR is good, but better if you can comment to explain what wallet is used for

Comment (cardanocasinovip) [reply src/tokens/30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea43484950.yaml:24]: @0xj4m35 this wallets is our updated treasury wallets for CHIP token

Comment (0xj4m35) [reply src/tokens/30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea43484950.yaml:24]: @cardanocasinovip I mean you can comment like this: 
image

Comment (cardanocasinovip) [reply src/tokens/30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea43484950.yaml:24]: `addr1qypez7msuwynwjm77vtrmmn9pm5v22y5th26exxe0kepst24k83f37z3s7fapxjamxkeedc085vg0y34zgn8x4c5m05q5nzhsy # reserved treasury 

addr1q9v5l0dmy6jaz9dwkww9ghtg3yzfr6js2pejwyp6m7tjtk87hpufs2s7rtta3x4tfwa79qfht3ccxhwd7p835jytpa4qf2y00d # staking rewards

addr1q90e9f6p69ad95998f5l9huuf70s3pd3m7yg9xynuypln3sr96ajdrxwdjdxrkayjn48qm2h35jxjdnsqlusfg027kyq97xapw # yield farming treasury

addr1q9nczapcdhg7yc23ysun3tjtkx2lydkjw76pem86dw7htu69sqkqf5r066h5ca2g5qldne4yn8r3hxzm73ez6urgss3sruu6l2 # platform incentives

addr1q8swgmuz39jpknze9q8ljmwq6ndse8wpp36d8xfcxf7vkj5qp8z9c4tw4tkxer2dlsejueexww9jdp867de9xqvw6ussslysgu # marketing

addr1q8v6xff20j88n7jekxw53x0g3pp7c2gcsx738q456hue6m0u9w6rhn6wgwqpsg24jfnk79lexhuzpmaj4rf9ghtw73jqmp0e04 # airdrop`

Comment (cardanocasinovip) [reply src/tokens/30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea43484950.yaml:24]: Is this better @0xj4m35 ?

Comment (0xj4m35) [reply src/tokens/30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea43484950.yaml:24]: @cardanocasinovip yeah, but please apply it into your PR

Comment (cardanocasinovip) [reply src/tokens/30d2ebdb2fec06142ee84e5120c2717b4d68a91bffd924420d94ddea43484950.yaml]: @0xj4m35 updated

**Review by 0xj4m35 — APPROVED** [review]

**Comment by cardanocasinovip** [comment]
Hi @0xj4m35 once I change the treasury address do I need to create another PR for updating the marketcap and circulating supple?

**Comment by 0xj4m35** [comment]
> Hi @0xj4m35 once I change the treasury address do I need to create another PR for updating the marketcap and circulating supple?

@cardanocasinovip correct.

**Comment by cardanocasinovip** [comment]
where can I update the marketcap and circulating supply, what file? @0xj4m35

**Comment by 0xj4m35** [comment]
> where can I update the marketcap and circulating supply, what file? @0xj4m35

@cardanocasinovip those info are calculated dynamically by info provided in `yaml` file. Pls read #README to understand

**Pull request merged**